release/BUILD

```diff
@@ -94,3 +121,11 @@ release_filegroup(
         "@io_k8s_release//:kubernetes-node-linux-amd64",
     ],
 )
+
+release_filegroup(
```

Comment:
Would it make sense to instead have a single release_filegroup that is multi-platform like done in https://github.com/kubernetes/kubernetes/blob/16b909ce1483258464715c9ab76b4dff82d596e5/build/BUILD#L132 ?

Reply:
Ah, that makes sense. I'll go ahead and change it.